builders/ubuntu-python-builder.psm1

```diff
@@ -31,7 +31,7 @@ class UbuntuPythonBuilder : NixPythonBuilder {
         $pythonBinariesLocation = $this.GetFullPythonToolcacheLocation()
 
         ### To build Python with SO, passing relative path W.r.t to the binary location.
-        $env:LDFLAGS="-Wl,-rpath='`$`$ORIGIN/../lib'"
+        $env:LIBS = "-Wl,--enable-new-dtags,-rpath='`$`$ORIGIN/../lib'"
```

Comment:
What's the benefit of LIBS over LDFLAGS?

Reply:
that's in the commit message:

> LDFLAGS applies to everything including later user builds (e.g. building a wheel)
LDFLAGS_NODIST could be used but is probably unnecessary (also applies to cpython stdlib C modules which is probably unwanted)
LIBS only applies to the python executable and libpython library, this is most likely enough. https://docs.python.org/3/using/configure.html#envvar-LIBS